docs/glossary.asciidoc

```diff
@@ -214,6 +214,13 @@ syslog, Apache, and other webserver logs. See
 [[k_glos]]
 == K
 
+[[glossary-kibana-privileges]] {kib} privileges ::
+// tag::kibana-privileges-def[]
+Enable administrators to grant users read-only, read-write, or no access to 
```

Comment:
```suggestion
Enables administrators to grant users read-only, read-write, or no access to 
```

Reply:
Per chat with @gchaps I'll disregard this suggestion and keep "kibana privileges... enable"